### ISSUE TITLE
Editing a preferences shows the value at loading time, not the current pref's value

### ISSUE THREAD

**Issue by antoine2711** [OpenRefine/OpenRefine#2768]
**Describe the bug**
When you edit a pref, the value shown in the prompt will always be the value at the page loaded time, and not the current value.

**To Reproduce**
1. Go to the Preference window
2. Create a pref, let's say « TestPref », and assign it `100`.
3. Reload the page, you can see your new pref with the good value
4. Edit the value of the « TestPref » to `TEXT`
5. Go to edit AGAIN the value of the « TestPref » and see the value `100` proposed, instead of `TEXT`, the current value of this preference.

**Screenshots**
Step 4.
![image](https://user-images.githubusercontent.com/12217525/85208674-45c2fd00-b300-11ea-8a4d-8705e25f5ba1.png)

Step 5.
![image](https://user-images.githubusercontent.com/12217525/85208697-786cf580-b300-11ea-8e49-f068e04c0035.png)

**Desktop:**
 - OS: Mac OS X.14
 - Browser Version: Chrome (current)

**OpenRefine :**
 - Version v3.3. to v3.5

**Comment by tfmorris** [comment]
@antoine2711 Please don't assign people to issues. Let them volunteer themselves. If you need a reviewer for a PR, Github has mechanisms to request that. If you need discussion, you can ping people like:
@thadguidry Did you agree to be assigned to this?

**Comment by thadguidry** [comment]
@tfmorris No sir.

**Comment by antoine2711** [comment]
> @antoine2711 Please don't assign people to issues. Let them volunteer themselves. If you need a reviewer for a PR, Github has mechanisms to request that. If you need discussion, you can ping people like:
> @thadguidry Did you agree to be assigned to this?

Hi @tfmorris: adding you here as an assigne was a mistake. I wanted to ask you to review the code that fixes this, and, of course, that in the PR, which I did after. 

And yes, I should not have added @thadguidry, but rather myself, which I just did. I created this issue but I also already had the code to fix it. So, I'm really looking to get this done from the start to the solution.

Regards,
   Antoine
